### PR TITLE
Extract hardcoded starting skill/spell/items and combat bonuses

### PR DESCRIPTION
**Pull request by ephphatha**
The low-hanging fruit of the areas kphoenix identified in #211. Based the split and naming convention off what was mentioned in discord (but written from memory so probably nowhere near the original).

Eventually aiming to load these values from CharacterAttributes.tsv, CharacterLoadouts.tsv, CharacterCombatStats.tsv (or something along those lines). D2 uses CharStats.txt for all of these combined which ends up being a massive 80 column wide file...

### CODE REVIEW

File: Source/playerdat.cpp [hunk 1]
```diff
@@ -152,6 +152,45 @@ void ReloadExperienceData()
...
+/* HeroClass::Sorcerer */  { N_("Sorcerer"),       15,      35,         15,        20,        45,    250,         85,        80,  (9 << 6),                    -(2 << 6),  (1 << 6),  (2 << 6), (1 << 6),                    (2 << 6),                    (1 << 6),                     (2 << 6), },
+/* HeroClass::Monk */      { N_("Monk"),           25,      15,         25,        20,       150,     80,        150,        80, (23 << 6),  static_cast<int>(5.5F * 64),  (2 << 6),  (2 << 6), (1 << 6),                    (1 << 6), static_cast<int>(1.5F * 64),  static_cast<int>(1.5F * 64), },
+/* HeroClass::Bard */      { N_("Bard"),           20,      20,         25,        20,       120,    120,        120,       100, (23 << 6),                     (3 << 6),  (2 << 6),  (2 << 6), (1 << 6), static_cast<int>(1.5F * 64), static_cast<int>(1.5F * 64), static_cast<int>(1.75F * 64), },
+/* HeroClass::Barbarian */ { N_("Barbarian"),      40,       0,         20,        25,       255,      0,         55,       150, (18 << 6),                     (0 << 6),  (2 << 6),  (0 << 6), (2 << 6),                    (1 << 6), static_cast<int>(2.5F * 64),                     (1 << 6), },
+	// clang-format on
```
Comment (DakkJaniels) [inline Source/playerdat.cpp:167]: Missing end of translation block

Comment (ephphatha) [reply Source/playerdat.cpp:168]: Thanks for the pickup, added to the commit where I move the block into the anonymous namespace (once I remembered it needs to go above the last string marked for translation...)

File: Source/utils/parse_int.hpp [hunk 2]
```diff
@@ -32,4 +36,30 @@ ParseIntResult<IntT> ParseInt(
 	return value;
 }
 
+inline uint8_t ParseFixed6Fraction(std::string_view str, const char **endOfParse = nullptr)
```
Comment (glebm) [inline Source/utils/parse_int.hpp:39]: "Base mana" can be negative or fractional, so it's probably not loaded correctly when it's negative with this code

Comment (ephphatha) [reply Source/utils/parse_int.hpp:39]: The integer part is loaded as a signed value, the fraction part is always positive though (it'll parse to a value in the range `[0, 1]`). I should add a test case to make sure since I don't think any of the existing test cases cover a class with negative starting/bonus mana.

Comment (glebm) [reply Source/utils/parse_int.hpp:39]: Oh I see, this only parses the fractional part. :+1:

**Comment by ephphatha** [comment]
Rebased on  #6608 the diff will shrink significantly when that is merged. I'm not looking forward to trying to load items from the per class files, but that's a problem for the future 😅

**Review by glebm — APPROVED** [review]

**Comment by AJenbo** [comment]
Super exciting with all the progress on this :)
